Lint checks failure

```
13:22:22 + python3 setup.py flake8 test
13:22:23 running flake8
13:22:24 WARNING: flake8 setuptools integration is deprecated and scheduled for removal in 4.x.  For more information, see https://gitlab.com/pycqa/flake8/issues/544
13:22:24 shukujitsu/__main__.py:27:80: E501 line too long (80 > 79 characters)
13:22:24 shukujitsu/__main__.py:28:80: E501 line too long (89 > 79 characters)
13:22:24 shukujitsu/__main__.py:29:80: E501 line too long (96 > 79 characters)
13:22:24 Build step 'Execute shell' marked build as failure
```

--- a/.github/workflows/flake8-pytest.yml
+++ b/.github/workflows/flake8-pytest.yml
@@ -34,6 +34,8 @@ jobs:
         flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
         # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
         flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+        # normal run
+        flake8 . --show-source
     - name: Test with pytest
       run: |
         pytest

--- a/shukujitsu/__main__.py
+++ b/shukujitsu/__main__.py
@@ -24,10 +24,12 @@ def getargs(args):
     jp_holidays = shukujitsu.Japan()
     parser = ArgumentParser(prog="shukujitsu",
                             formatter_class=RawTextHelpFormatter,
-                            description="Select Japanese holidays from %d to %d"
-                                        % (min(jp_holidays).year, max(jp_holidays).year),
-                            epilog="With no DATE, read standard input. Exit status is 0 if any "
-                                   "date is selected, 1 otherwise.")
+                            description="Select Japanese holidays from %d to "
+                                        "%d" % (min(jp_holidays).year,
+                                                max(jp_holidays).year),
+                            epilog="With no DATE, read standard input. Exit "
+                                   "status is 0 if any date is selected, 1 "
+                                   "otherwise.")
     parser.add_argument("-i", "--invert-match", action="store_true",
                         help="select non-holidays")
     parser.add_argument("-n", "--holiday-name", action="store_true",

--- a/test_shukujitsu.py
+++ b/test_shukujitsu.py
@@ -34,7 +34,7 @@ class HolidaysTest(TestCase):
     @wrap_case(datetime(2022, 5, 4), True)
     @wrap_case(datetime(2022, 5, 5), True)
     @wrap_case(datetime(2023, 5, 5), True)
-    @wrap_case(datetime(2024, 5, 5), False)  # Not yet announced holidays in 2024
+    @wrap_case(datetime(2024, 5, 5), False)  # No data for holidays in 2024
     def test_holidays(self, day, expect):
         if expect:
             self.assertIn(day, JapaneseHolidays())
@@ -109,11 +109,12 @@ class DataTest(TestCase):
 
 
 class SetupPyTest(TestCase):
+    classifier = "Programming Language :: Python :: %s.%s" % version_info[:2]
+
     def test_classifiers_version(self):
         here = dirname(abspath(__file__))
         with open(join(here, "setup.py")) as fp:
-            classifier = "Programming Language :: Python :: %s.%s" % version_info[:2]
-            self.assertIn(classifier, fp.read())
+            self.assertIn(SetupPyTest.classifier, fp.read())
 
 
 if __name__ == "__main__":

